README.md

```diff
@@ -63,8 +63,11 @@ export PATH=${PATH}:/Applications/Postgres.app/Contents/Versions/11/bin/
 
 To switch redis on you'll need to install it locally. On a OSX we've used brew for this. To use redis caching you need to switch it on by changing the config for development:
 
-        REDIS_ENABLED = True
+```
+REDIS_ENABLED = 1
```

Comment:
I'm confused. Is the suggestion here to do

`REDIS_ENABLED=1 make run-celery-with-docker`

or is it to go and change the app/config.py file and set `REDIS_ENABLED = 1`? 

If it is the former, then we would need to remove the space characters here and change the description of the comment above which talks about changing the development config which I read as related to app/config.py.

Or if it is the later, then I'm not quite sure why we need to change it to `1` as I feel that `True` would also turn redis on when editing the app/config.py directly?

Reply:
Ahhh, so _that's_ what it was referring to.

You're right about the spaces. I've amended in https://github.com/alphagov/notifications-api/pull/3467/commits/6092baea643e5058237fd0d48e06f9d8f5918a65.